catalog/spatial/wfs/1.1.0/spatial-wfs-v1_1_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v110/catalog/source/WfsFilterDelegate.java

```diff
@@ -719,9 +698,6 @@ private DateTime convertDateToIso8601Format(Date inputDate) {
   // spatial operators
   @Override
   public FilterType beyond(String propertyName, String wkt, double distance) {
-    if (!isEpsg4326) {
-      return null;
-    }
```

Comment:
Don't we need to do something that tells the converter what kind of encoding is being used?

Reply:
no, because the encoding should always be 4326.